### PR TITLE
Movement in Rank doughnut complete

### PR DESCRIPTION
**Pull request by deanajackson**
I used a combo of Linh's structure for the visuals and the previous logic I had for the "weekly movement in rank" visual and produced a second visual for the dashboard.

### CODE REVIEW

File: src/components/ListMovementDonut.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+function generateRandomColors() {
```
Comment (sieday) [inline src/components/ListMovementDonut.jsx:21]: Lets standardize the coloring across all of the components, is there a way to pull it into a separate styling file for all of our graphical components?

File: src/components/ListMovementDonut.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    const g = Math.floor(Math.random() * 256);
+    const b = Math.floor(Math.random() * 256);
+    const alpha = 0.5;
+    return `rgba(${r}, ${g}, ${b}, ${alpha})`;
+  });
+}
+
+const ListMovementDoughnut = (props) => {
+  const getNewData = () => {
+    let keyToSearch = "list_name";
```
Comment (sieday) [inline src/components/ListMovementDonut.jsx:33]: String has double quotes instead of single - Airbnb?

**Review by sieday — APPROVED** [review]
Added a few small comments, but looks great overall!

**Review by linh224 — COMMENTED** [review]
Hi,
I think everything is good now. We can continue to make our dashboard more visual :)

**Pull request merged**